apps/channels/tests/test_base_channel_behavior.py

```diff
@@ -115,8 +116,10 @@ def test_different_sessions_created_for_different_users(_get_llm_response, teleg
     experiment_sessions_count = ExperimentSession.objects.count()
     assert experiment_sessions_count == 2
 
-    assert ExperimentSession.objects.filter(external_chat_id=user_1_chat_id).exists()
-    assert ExperimentSession.objects.filter(external_chat_id=user_2_chat_id).exists()
+    assert ExperimentSession.objects.filter(participant__external_chat_id=user_1_chat_id).exists()
+    assert Participant.objects.get(external_chat_id=user_1_chat_id)
```

Comment:
This 2nd assert is necessary since the one above would fail if the participant didn't exist